apps/web/wrangler.toml

```diff
@@ -26,22 +26,22 @@ bucket_name = "dev-r2-anycontext"
 
 [[d1_databases]]
 binding = "DATABASE"
-database_name = "dev-d1-anycontext"
-database_id = "fc562605-157a-4f60-b439-2a24ffed5b4c"
+database_name = "supermemory-db-preview"
```

Comment:
for local development only this database is required, you do not need to create the other two databases